missas/core/tests/test_city_queryset.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from model_bakery import baker
+
+from missas.core.models import City
+
+
+@pytest.mark.django_db
+class TestFilterWithSchedule:
+    def test_returns_cities_with_schedules(self):
+        city_with_schedule = baker.make("core.City")
+        parish_with_schedule = baker.make("core.Parish", city=city_with_schedule)
+        baker.make("core.Schedule", parish=parish_with_schedule)
+
+        city_without_schedule = baker.make("core.City")
+        baker.make("core.Parish", city=city_without_schedule)  # Parish but no schedule
+
+        result = City.objects.filter_with_schedule()
+
+        assert city_with_schedule in result
+        assert city_without_schedule not in result
+        assert result.count() == 1
+
+    def test_with_multiple_cities_with_schedules(self):
+        # City 1 with schedule
+        city1 = baker.make("core.City")
+        parish1 = baker.make("core.Parish", city=city1)
+        baker.make("core.Schedule", parish=parish1)
+
+        # City 2 with schedule
+        city2 = baker.make("core.City")
+        parish2 = baker.make("core.Parish", city=city2)
+        baker.make("core.Schedule", parish=parish2)
+
+        # City 3 without schedule
+        city3 = baker.make("core.City")
+        baker.make("core.Parish", city=city3)
```

Comment:
Do not use these comments. Use variable name that auto-explains. Rename city3 to city_without_schedule. Do it in the entire file.